lib/openstax/exchange/client/fake_client.rb

```diff
@@ -0,0 +1,77 @@
+require 'securerandom'
+require 'uri'
+
+module OpenStax
+  module Exchange
+    module Client
+      class FakeClient
+        include ClientInstance
```

Comment:
This includes two methods that `raise NotImplementedError`.  Why is this useful?